src/pages/newBlogpost/newBlogpostPage.jsx

```diff
@@ -0,0 +1,65 @@
+import './newBlogpostPage.css'
+import {useForm} from "react-hook-form";
+
+function NewBlogpost() {
+    const {register, handleSubmit} = useForm();
+
+    function handleFormSubmit(data){
```

Comment:
In de handleFormSubmit krijg je via de data parameter een object binnen. Aan een object kun je gemakkelijk nieuwe attributes en values toevoegen. Dit kun je bijvoorbeeld doen door:
```
data.shares = 0;
data.comments = 0;
```
Als je daarna weer een console.log(data) zou uitvoeren zou je nu de nieuwe attributen met bijbehorende values moeten zien.